app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.java

```diff
@@ -417,6 +389,38 @@ private int howManyItemsToLoad() {
         return Math.max(MIN_ITEMS_INITIAL_LOAD, items);
     }
 
+    @Nullable
+    private String tryRetrieveVideoIdFromUrl(String url) {
+        try {
+            return youtubeStreamLinkHandler.getId(url);
+        } catch (Exception ex) {
+            if (DEBUG)
+                Log.d(TAG, "Failed to retrieve video ID from \"" + url + "\"", ex);
+
+            showSnackBarError(ex,
+                UserAction.SOMETHING_ELSE,
+                "none",
+                "",
```

Comment:
Here you should put `url`, so that the chash report contains the URL in the field "Request: " and it is simpler to debug